README.md

```diff
@@ -34,6 +34,7 @@ This package provides an implementation of four classic algorithms for unconstra
     > high-order regularized models. *Mathematical Programming*, 163(1), 359-368.
     > DOI: [10.1007/s10107-016-1065-8](https://doi.org/10.1007/s10107-016-1065-8)
 
+- `R2N`: an inexact second-order quadratic regularization method for unconstrained optimization with shifted L-BFGS systems;
```

Comment:
Where does the name R2N comes from?

Reply:
The name is to show that there is a Newton (quasi newton method) as we add B which usually is a LBFGS method 
open to suggetions